### PR TITLE
Attempted to address Issue #4978

### PR DESCRIPTION
**Pull request by tnwoodard**
I installed simple forms to generate a form in the FAQ section for updating student roster.

NOTE: Please review the pull request process before opening your first PR: https://github.com/WikiEducationFoundation/WikiEduDashboard/blob/master/CONTRIBUTING.md#pull-request-process

## What this PR does
The purpose of this PR is to address Issue #4978  

## Screenshots
Before:
< add a screenshot of the UI before your change >
No UI available. This is a new enhancement.

After:
< add a screenshot of the UI after your change >

FAQs with link to update roster
Form to update roster with update user clicked

## Open questions and concerns
I have some questions and concerns listed below:
1. Where is the information going?  Once the "Update User" button is clicked, the new username will be added where?
2. Is there more to be done with `UserImporter.update_username_for_global_id`?
3. Currently there is a link to update the user on the FAQ page ('/faq'). It would be nice to add it to Activity section or somewhere else that is easy to find.
4. Eventually, the form will need some style.

Thank you,
Tempestt

### CODE REVIEW

File: config/initializers/simple_form_bootstrap.rb [hunk 1]
```diff
@@ -0,0 +1,440 @@
+# frozen_string_literal: true
```
Comment (ragesoss) [inline config/initializers/simple_form_bootstrap.rb:1]: I don't think adding this is necessary. To make the feature as simple and maintainable as possible, it should use existing UI patterns and follow the conventions of the rest of the codebase; I don't want to add a large amount of initializer code like this that I don't fully understand.

File: config/locales/en.yml [hunk 2]
```diff
@@ -59,7 +59,7 @@ en:
     log_out: Log out
     my_dashboard: My Dashboard
     recent_activity: Recent Activity
-    report_problem: Report a problem
+    report_problem:
```
Comment (ragesoss) [inline config/locales/en.yml:62]: Looks like this was inadvertant?

File: app/views/faq/index.html.haml [hunk 3]
```diff
@@ -24,3 +24,5 @@
         All questions
       - else
         %a.topic-link{href: "/faq?all=true"} All questions
+        %br
+        %a.topic-link{href: "/update_roster"} Update student roster
```
Comment (ragesoss) [inline app/views/faq/index.html.haml:28]: I don't think we need a hard-coded link to this from the FAQ index; linking to it from an individual FAQ is something we can do from the frontend (by creating or editing a relevant FAQ entry) after the form is deployed.

File: app/controllers/roster_controller.rb [hunk 4]
```diff
@@ -0,0 +1,13 @@
+class RosterController < ApplicationController
```
Comment (ragesoss) [inline app/controllers/roster_controller.rb:1]: I don't think "roster" is the right name to use here. This controller's job will be to trigger a username update, which is independent of which courses that user is participating in. (We also don't use the term "roster" anywhere else in the system, so it's likely to be confusing.)

File: app/controllers/roster_controller.rb [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+  before_action :require_signed_in
+  respond_to :html, :json
+  def update_roster
```
Comment (ragesoss) [inline app/controllers/roster_controller.rb:4]: Where is the code that actually updates the `User` record?

**Comment by ragesoss** [comment]
This should include a Feature spec to verify that it works as intended, updating a `User` record with that user's current username if the Dashboard currently has an outdated username for that account.

**Comment by tnwoodard** [comment]
Thanks for the feedback, Sage. I will work on this to address your
feedback.


-Tempestt

File: config/initializers/simple_form_bootstrap.rb (hunk 1 above)
Comment (tnwoodard) [reply config/initializers/simple_form_bootstrap.rb:1]: Ok, I will remove Bootstrap UI.

File: app/views/faq/index.html.haml (hunk 3 above)
Comment (tnwoodard) [reply app/views/faq/index.html.haml:28]: Oh ok. Do you have a preference of where the form should be located?

File: config/locales/en.yml (hunk 2 above)
Comment (tnwoodard) [reply config/locales/en.yml:62]: Yes. I'm unsure how it happened, but I added it back.

File: app/controllers/roster_controller.rb (hunk 5 above)
Comment (tnwoodard) [reply app/controllers/roster_controller.rb:4]: It will go in the enrollment_controller.rb file.

File: app/views/faq/index.html.haml (hunk 3 above)
Comment (ragesoss) [reply app/views/faq/index.html.haml:28]: Somewhere similar, perhaps `/update_username`, would make sense.

**Comment by tnwoodard** [comment]
Hi Sage,

I tried to address the comments with `update_username` and creating a Feature spec. I learned about test specs in my bootcamp program, but this is my first test spec. I really appreciate your feedback. Thank you.

File: app/views/faq/index.html.haml [hunk 6]
```diff
@@ -19,8 +19,3 @@
         = topic.name
       - else
         %a.topic-link{href: "/faq?topic=#{topic.slug}"}= topic.name
-    %hr
```
Comment (ragesoss) [inline app/views/faq/index.html.haml:22]: It looks like you've removed this by accident.

**Pull request closed without merging**